pact/policy-manager/policy-manager.pact

```diff
@@ -263,14 +267,14 @@
 
     (if (exists-quote sale-id)
```

Comment:
The `if` and `else` branches are almost identical here. We can move the calls to `policy::enforce-withdraw` outside the if statement so that code isn't duplicated.
Also I think we're missing a call to `(close-quote sale-id)`